x-pack/plugins/security/public/plugin.ts

```diff
@@ -28,18 +43,48 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     this.sessionTimeout = new SessionTimeout(notifications, sessionExpired, http, tenant);
     http.intercept(new SessionTimeoutHttpInterceptor(this.sessionTimeout, anonymousPaths));
 
+    this.license$ = licensing.license$;
+
     return {
       anonymousPaths,
       sessionTimeout: this.sessionTimeout,
     };
   }
 
-  public start() {
+  public start(core: CoreStart) {
     this.sessionTimeout.start();
+
+    const securityLicenseService = new SecurityLicenseService().setup();
```

Comment:
question: we're calling `SecurityLicenseService::setup` from within the plugin's `start` lifecycle method, shouldn't we be calling `setup` from within the `setup` lifecycle event?

Reply:
I originally had it there, but we can't take advantage of the service until the `start` lifecycle, and it felt silly to turn this into an instance variable just so it could be shared across the two lifecycles. The `SecurityLicenseService::setup` isn't doing anything noteworthy, so it felt safe to use it in `start` like this.

With the introduction of a dedicated nav control service, this becomes a moot point anyway